docs/deployment_and_config.md

```diff
@@ -283,6 +283,27 @@ Replace the entries in `<>` with your values:
 - `custom_cert.cert`: path to your `.crt` or `.pem` certificate
 - `custom_cert.key`: path to your `.key` file
 
+Sometimes, data sources such as `git` or `gerrit` would require access using
```

Comment:
You missed to add the title to this section: `#### SSH Keys for Non-Public Data Sources (Optional)`